keps/sig-node/20190717-seccomp-ga.md

```diff
@@ -262,6 +318,28 @@ type SeccompProfileSet struct {
 
 ## Design Details
 
+
+### Failure and Fallback Strategy
+
+There are different scenarios in which applying seccomp may fail, below are the ones we mapped 
+and their outcome once this KEP is implemented:
+
+| Scenario | API Server Result | Kubelet Result |
+|--------------------------------------------------------------|------------------------------|----------------------------------------------------------------------------------------------|
+| 1) Using custom or `runtime/default` profile when container runtime does not support seccomp. | Pod created | The outcome is container runtime dependent. In this scenario containers may 1) fail to start or 2) run normally without having its policies enforced. |
+| 2) Using custom or `runtime/default` profile that restricts syscalls a container is trying to make. | Pod created | The outcome is workload and seccomp dependent. In this scenario containers may 1) fail to start, 2) misbehave or 3) log violations. |
+| 3) Using custom profile that relies on unsupported or invalid seccomp actions (i.e. `SCPM_ACT_LOG` in versions earlier than 4.14 Linux kernel). | Pod created | Containers fail to start. Retry respecting RestartPolicy and back-off delay. |
```

Comment:
Are these failures visible in the container log? In events logged by the kubelet?